c-blosc/blosc/shuffle.c

```diff
@@ -123,10 +123,9 @@ static uint64_t _xgetbv(uint32_t xcr) {
     which means we can get away with returning a hard-coded value from
     this implementation of _xgetbv. */
 
-static inline uint64_t
-_xgetbv(uint32_t xcr) {
+static __inline uint64_t _xgetbv(uint32_t xcr) {
     /* A 64-bit OS must have XMM save support. */
-    return xcr == 0 ? (1UL << 1) : 0UL;
+    return (xcr == 0 ? (1UL << 1) : 0UL);
```

Comment:
Without the patch building fails on  MSVC2008 (=Visual C++ 9.0), which is used for CPython 2.7.

It's an inline defined for x64 targets with MSVC2010 SP1. So it might fail on MSVC2010 as well, but I haven't tested it.

I found the fix in the anaconda (and conda-forge) pytables recipe.